tests/foreman/ui/test_jobtemplate.py

```diff
@@ -158,6 +161,7 @@ def test_positive_end_to_end(session, module_org, module_loc):
                 == job_foreign_input_sets[1]['include_all'])
         assert (template['job']['foreign_input_sets'][1]['include']
                 == job_foreign_input_sets[1]['include'])
+        session.organization.select(org_name="Default Organization")
```

Comment:
What is the reason for changing organization in the middle of the test?

Reply:
In Sat 6.7 is host needed to show template preview. I use satellite itself as host to show this template. This host is not in created organization so I need to change it.

Reply:
Makes sense. Thanks for explanation